clang/include/clang/Sema/Sema.h

```diff
@@ -5146,6 +5153,12 @@ class Sema final : public SemaBase {
     return ExprEvalContexts.back();
   };
 
+  const ExpressionEvaluationContextRecord &parentEvaluationContext() const {
+    assert(ExprEvalContexts.size() >= 2 &&
+           "Must be in an expression evaluation context");
+    return ExprEvalContexts[ExprEvalContexts.size() - 2];
```

Comment:
I can grep at least 4 places in clang/lib/sema where we do `ExprEvalContexts.size() - 2`. Would be a good first issue to switch these to `parentEvaluationContext` calls (once we merge this patch)?

Reply:
Yes, I was actually thinking of doing that!

Reply:
Submitted https://github.com/llvm/llvm-project/issues/93284